auth.go

```diff
@@ -474,32 +435,87 @@ func (x *Central) Login(identity, password string) (sessionkey string, token *To
 }
 
 // Merges ldap with user store every 10 seconds
-func (x *Central) Merge() error {
+func (x *Central) MergeTicker() error {
 	x.Log.Info("Starting LDAP merge process")
-	x.Ticker = time.NewTicker(time.Second * 10)
+	x.Ticker = time.NewTicker(ldapMergeTickerSeconds)
```

Comment:
This function should be named "StartMergeTicker" so make it clear that it is launching a process that is going to run for some time. In our case, indefinitely.

The control flow of this function is more complicated than it needs to be. You can just do
```go
go func() {
    x.Ticker := time.NewTicker(ldapMergeTickerSeconds)
    for _ = range x.Ticker.C {
        // do stuff
    }
}()
```

I don't think we should emit log messages stating that we are about to do a merge. Only emit log messages when something changes, or when an error occurs. Otherwise our log files end up with a million "starting merge/finishing merge" messages, which don't add any signal to the noise.
It is however useful to have timing information about the merge process. What I would do is to have a counter that increments every time a merge occurs. Only if that counter mod 60 equals zero, then emit the time. That way, we get one timing message per hour, which is reasonable. Also, count the time duration in the program, and emit that directly into the logs (instead of emitting two absolute time measurements).

I don't think that the ticker should abort if it encounters an error. If it encounters an error, it should log that failure, and then proceed as usual. Think about what happens if we abort on error. It means that the merge ticker kills itself, and so DevOps has to logon to that server and restart the auth service, which is definitely not what we want.